common4j/src/main/com/microsoft/identity/common/java/foci/FociQueryUtilities.java

```diff
@@ -129,9 +132,15 @@ public static boolean tryFociTokenWithGivenClientId(@SuppressWarnings(WarningTyp
         //       for every FoCI apps (and hardcode it here).
         //       https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1222002
         if (ObjectUtils.equals(clientId, "87749df4-7ccf-48f8-aa87-704bad0e0e16")) {
-            scopes = "https://devicemgmt.teams.microsoft.com/.default " + BaseController.getDelimitedDefaultScopeString();
-            Logger.info(TAG + methodName,
-                    "Teams agent client ID - making a test request with teams agent resource.");
+            final Span span = GlobalOpenTelemetry.getTracer(TAG).spanBuilder("setScopeForDMAgentForFoci").startSpan();
```

Comment:
`setScopeForDMAgentForFoci` can be defined in `SpanName.java`

Reply:
SpanName.java is present in broker. Shall I move it to common?

Reply:
Yes, feel free to do that. Or it is also fine the way it is and we can move it to common later once we have more spans in common.